### PR TITLE
feat: add basic version of lacchain board

### PR DESCRIPTION
**Pull request by adriexnet**
### What does this PR do?

- add first version for lacchain board
- allow run using different environments with `make run environmentName`

### Steps to test

1. make run local
2. navigate to http://localhost:3000

#### CheckList

- [x] Follow proper Markdown format
- [x] The content is adequate
- [x] The content is available in both english and spanish
- [x] I Ran a spell check

### CODE REVIEW

**Review by xavier506 — CHANGES_REQUESTED** [review]
Nice! 

Please see CF comments

File: hapi/src/workers/producers.worker.js [hunk 1]
```diff
@@ -47,17 +47,17 @@ const start = async () => {
...
-    producerService.syncProducersInfo,
-    workersConfig.syncProducerInfoInterval
-  )
-  run(
-    'SYNC CPU USAGE',
-    producerService.syncCpuUsage,
-    workersConfig.syncProducerCpuInterval
-  )
-  run('CHECK FOR MISSED BLOCK', producerService.checkForMissedBlocks)
+  // run(
```
Comment (xavier506) [inline hapi/src/workers/producers.worker.js:50]: @adriexnet why is this commented?

Comment (adriexnet) [reply hapi/src/workers/producers.worker.js]: My mistake, it was fixed in the las commit

**Comment by xavier506** [comment]
@adriexnet  is this ready to be merged?

**Pull request merged**